_posts/2025-04-03-geoserver-2-27-0-released.md

```diff
@@ -0,0 +1,186 @@
+---
+author: Gabriel Roldan
+date: 2025-04-03
+layout: post
+title: GeoServer 2.27.0 Release
+categories:
+- Announcements
+- Vulnerability
+tags:
+- Release
+release: release_227
```

Comment:
We need to make this, based on [release_226.html](https://github.com/geoserver/geoserver.github.io/blob/main/_layouts/release_226.html)